### PR TITLE
Add code coverage with blanket

### PR DESCRIPTION
**Pull request by madjid04**

### CODE REVIEW

**Comment by madarche** [comment]
@zaach could you accept this PR please? I would rather move from Istanbul to Blanket. The next step is to have a CI generate the coverage % from blanket output. Thanks!

File: package.json [hunk 1]
```diff
@@ -37,9 +38,14 @@
...
     "test": "jshint *.json lib/*.js test/*.js test/cases/*.js test/cases/*.json && mocha --check-leaks -R spec",
-    "test-cov": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter dot --check-leaks test/",
+    "test-cov": "mocha --require blanket -R html-cov > test/coverage.html",
     "test-travis": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -- --reporter spec --check-leaks test/"
   },
+  "config": {
+   "blanket": {
+    "pattern": "lib/convict.js"
+  }
+},
```
Comment (zaach) [inline package.json]: Nit: the indentation is off on the four lines above.

**Comment by madarche** [comment]
@zaach right you were. Sorry for not having spotted 2 errors. With the last commit the PR seems correct to me now. Thank you.

**Comment by zaach** [comment]
Thanks :+1:

**Comment by madarche** [comment]
Thanks a lot @zaach

**Pull request merged**